app/Http/Controllers/AdminController.php

```diff
@@ -1426,16 +1426,7 @@ public function monitor(): View
     {
```

Comment:
Since we now have a dedicated `MonitorController` class, perhaps this function should be moved there.